customize.sh

```diff
@@ -0,0 +1,12 @@
+ui_print "- Searching in fonts.xml"
+[[ -d /sbin/.magisk/mirror ]] && MIRRORPATH=/sbin/.magisk/mirror || unset MIRRORPATH
+FILE=/system/etc/fonts.xml
+FILENAME=$(sed -ne '/<family lang="und-Zsye".*>/,/<\/family>/ {s/.*<font weight="400" style="normal">\(.*\)<\/font>.*/\1/p;}' $MIRRORPATH$FILE)
+for i in $FILENAME
+do
+    ui_print "- Copying fonts files to $i"
+    cp -f $MODPATH/system/fonts/Blobmoji.ttf $MODPATH/system/fonts/$i
+done
+
+ui_print "- Setting permissions"
+set_perm_recursive $MODPATH 0 0 0755 0644
```

Comment:
Unnecessary.
https://github.com/topjohnwu/Magisk/blob/master/scripts/module_installer.sh#L132

Reply:
It depends, we do copy some files after that happens. But yeah, since they should keep the permissions of the original file, this can be removed.

Reply:
No I mean it's duplicate with the installer script. Delete it.

Reply:
Yeah, that's what I mean. I removed it in a commit just a few minutes ago.